tests/test_util.py

```diff
@@ -3,28 +3,7 @@
 import numpy as np
 import pytest
 from stmtools import utils
-
-@pytest.fixture
-def stmat():
-    npoints = 10
-    ntime = 5
-    return xr.Dataset(
-        data_vars=dict(
-            amplitude=(
-                ["points", "time"],
-                da.arange(npoints * ntime).reshape((npoints, ntime)),
-            ),
-            phase=(
-                ["points", "time"],
-                da.arange(npoints * ntime).reshape((npoints, ntime)),
-            ),
-        ),
-        coords=dict(
-            lon=(["points"], da.arange(npoints)),
-            lat=(["points"], da.arange(npoints)),
-            time=(["time"], np.arange(ntime)),
-        ),
-    ).unify_chunks()
+from .test_stm import stmat
```

Comment:
```suggestion
from tests.test_stm import stmat
```

Reply:
Pytest returns an error with relative import, so I suggest here adding `tests.`, can you check it?

Reply:
I found `tests.` also works, will fix this